### ISSUE TITLE
Do global options really need to be required?

### ISSUE THREAD

**Issue by jwhitlark** [l3nz/cli-matic#35]
Can just specify it as empty, but it can take a few minutes to figure out what the problem is, especially for those less familiar with spec.

**Comment by l3nz** [comment]
While we work on that, if the are missing or empty, it makes no sense to print them out:

```
USAGE:
 eml [global-options] command [command options] [arguments...]

```
but 

```
USAGE:
 eml  command [command options] [arguments...]

```

**Comment by thiru** [comment]
It's common for terminal apps to be called like `app --version` to print the version.  Is this possible?

**Comment by l3nz** [comment]
So closed in 0.3.9
